tools/tplgtool2.py

```diff
@@ -1330,6 +1330,7 @@ def main():
         errors = 0
         for f in files:
             tplg = GroupedTplg(tplgFormat.parse_file(f))
+            assert set(dump_types) <= set(supported_dump), f"unsupported type in {dump_types}"
```

Comment:
no idea what this does.

Reply:
Before this commit, a typo in the command-line argument was silently producing no output.